curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f04ed0035f47ed04d0f1f.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 657f04ed0035f47ed04d0f1f
+title: Step 13
+challengeType: 20
+dashedName: step-13
+---
+
+# --description--
+
+Before running the `main` function, you need to make sure that the file is running as a script. Add an `if` statement on the same level as the two existing function and check whether `__name__ == '__main__'` evaluates to `true` or not.
```

Comment:
```suggestion
Before running the `main()` function, you need to make sure that the file is running as a script. Add an `if` statement on the same level as the two existing functions and check whether `__name__ == '__main__'` evaluates to `True` or not.
```
I would suggest replacing `true` with `True` because in Python, truth values start with an uppercase letter.